scripts/ariitk_pkg

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+git clone --branch scripts git@github.com:rohanblueboybaijal/ariitk_template.git ~/.ariitk/ > /dev/null 2>&1
```

Comment:
won't you merge this into master? so will have to redirect this

Reply:
So it would be 
```git clone git@github.com:rohanblueboybaijal/ariitk_template.git ~/.ariitk/ > /dev/null 2>&1```?
@varunk122 
We'll change this once we merge into master